packages/ombits-client/src/app.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useCurrentRoute, useViewElement } from 'react-navi';
+import { animated, useTransition } from 'react-spring';
+
+export const App = () => {
```

Comment:
@remotealex is this App component and everything the way it's laid out is from react-navi example ? if so, can u share the link bro

Reply:
https://frontarm.com/navi/en/integrations/react-spring/